libraries/pbrlib/pbrlib_defs.mtlx

```diff
@@ -417,4 +417,75 @@
     <output name="extinction" type="color3" />
   </nodedef>
 
+  <!-- ======================================================================== -->
+  <!-- OpenPBR Surface Nodes backported from 1.39                               -->
+  <!-- ======================================================================== -->
```

Comment:
Need these 4 new nodes in PBR lib. Would be of interest to provide an upgrade path in case they get re-used somewhere else in client code. Upgrade path is to use the refreshed 1.39 node with proper input values for the requested mode.